pkg/metrics/acquire/acquire.go

```diff
@@ -99,6 +100,9 @@ func ReadLinesOffsetN(filename string, offset uint, n int) (ret []string, err er
 		// on EOF we also break, but we should have results
 		// on other errors, we should not have any result which we check on the "client"
 		if err != nil {
+			if err == io.EOF && len(line) > 0 && i >= int(offset) {
```

Comment:
I would add a comment to explain why this if exists ☝️ (i.e. ~ `read last line if no new line`)